### ISSUE TITLE
Cannot navigate between vim windows on OSX

### ISSUE THREAD

**Issue by tieubao** [christoomey/vim-tmux-navigator#127]
<img width="634" alt="screen shot 2016-05-21 at 3 51 40 pm" src="https://cloud.githubusercontent.com/assets/1749624/15447431/74b7dff8-1f6c-11e6-8a6b-1d42f8463f02.png">

I already install vim-tmux-navigator plugin, but I **can't use the hotkey** to navigate around. Using `:TmuxNavigateLeft` ... is good.

Everything is ok outside of tmux.

My env:

- Mac OSX
- Outside of TMUX: `echo $TERM` ---> xterm-256color
- Inside of TMUX: `echo $TERM` ---> screen-256color

**Comment by blueyed** [comment]
Does it affect all mapping, or only some of them?

**Comment by tieubao** [comment]
@blueyed right now, only for navigation mappings

**Comment by blueyed** [comment]
@tieubao 
So for all mappings from tmux-navigator?  (I was referring to if e.g. only `C-h` would not work)

**Comment by tieubao** [comment]
@blueyed all C-l, C-h, C-j, C-k

**Comment by blueyed** [comment]
Please try if https://github.com/christoomey/vim-tmux-navigator/pull/117 helps.
Also check that you've setup tmux correctly (which is different in #117): `tmux list-keys | grep -i vim`.

**Comment by tieubao** [comment]
@blueyed I added those lines in `.tmux.conf`

```
bind -n C-h run "(tmux display-message -p '#{pane_title}' | grep -iq vim && tmux send-keys C-h) || tmux select-pane -L"
bind -n C-j run "(tmux display-message -p '#{pane_title}' | grep -iq vim && tmux send-keys C-j) || tmux select-pane -D"
bind -n C-k run "(tmux display-message -p '#{pane_title}' | grep -iq vim && tmux send-keys C-k) || tmux select-pane -U"
bind -n C-l run "(tmux display-message -p '#{pane_title}' | grep -iq vim && tmux send-keys C-l) || tmux select-pane -R"
```

**Comment by blueyed** [comment]
Yeah, you would need to debug it from there, e.g. by adding some `display-message` there etc.
But I suggest trying this based on #117.

**Comment by fuadsaud** [comment]
been experiencing the same recently

**Comment by oblitum** [comment]
I'm running Vim on _Bash On Windows_ (WSL) and I'm having the same issue there too.

**Comment by Lucasosf** [comment]
I got the exact same problem as @tieubao oO

**Comment by thomascarreau** [comment]
Same issue over here.

**Comment by oblitum** [comment]
> I'm running Vim on Bash On Windows (WSL) and I'm having the same issue there too.

I'm now running [wsl-terminal](https://github.com/goreliu/wsl-terminal) (based on [mintty](http://mintty.github.io/)) and have no issues anymore :-)

**Comment by mschwartz** [comment]
I am experiencing this now.

latest iterm2, neovim installed from homebrew.

.tmux.conf:
```
 ~  cat ~/.tmux.conf
set-environment -g PATH "/usr/local/bin:/usr/bin"
# bind leader/prefix to C-a instead of C-b
unbind C-b
set -g prefix C-a
bind C-a send-prefix
```

^A r shows
```
~/.tmux/plugins/tpm/tpm returned 127
```
